src/servers/ZoneServer2016/classes/character.ts

```diff
@@ -56,5 +56,6 @@ export class Character2016 extends Character {
         slotId: 4,
       },
     ];
+    this.loadout = []
```

Comment:
if it's always empty at start, defined it like this `loadout: any[] = [];` before the constructor